CommentingPractice/comment-practice.js

```diff
@@ -84,15 +81,17 @@ import ColorConverter from './services/colorConverter';
      let temp = new Template("ROYBGIV","The colors of the rainbow in their respective order");
      console.log(temp.name + ": " + temp.description);
 
-    
+
     var templateObj = new Template("Martin", "First Commit")
     console.log(templateObj.name); 
 
+    let bear = new Animal("bear", "mammal", "omnivore");
+    console.log(bear.getName() + " is a " + bear.getGroup());
+
```

Comment:
If you take a look at the animal file, you'll see that doesn't have a constructor that takes arguments. So, the name property of the bear never is set. You would have to set the name and group properties.

Reply:
Shoot sorry I didn't realize that, I fixed it